Debug information for file: javali_tests/HW2/OkSimpleReturn.javali

Test failed because an exception was thrown:
    null
Stack trace:
cd.ToDoException
	at cd.frontend.parser.JavaliAstVisitor.visitClassDecl(JavaliAstVisitor.java:17)
	at cd.frontend.parser.JavaliAstVisitor.visitClassDecl(JavaliAstVisitor.java:1)
	at cd.frontend.parser.JavaliParser$ClassDeclContext.accept(JavaliParser.java:113)
	at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visitChildren(AbstractParseTreeVisitor.java:72)
	at cd.frontend.parser.JavaliBaseVisitor.visitUnit(JavaliBaseVisitor.java:24)
	at cd.frontend.parser.JavaliParser$UnitContext.accept(JavaliParser.java:66)
	at org.antlr.v4.runtime.tree.AbstractParseTreeVisitor.visit(AbstractParseTreeVisitor.java:44)
	at cd.Main.parse(Main.java:92)
	at cd.test.AbstractTestAgainstFrozenReference.testParser(AbstractTestAgainstFrozenReference.java:98)
	at cd.test.AbstractTestAgainstFrozenReference.test(AbstractTestAgainstFrozenReference.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
